content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -61,6 +64,35 @@ public final class ExampleChainableState extends ChainableState<ExampleChainable
 }
 ```
 
+{{% /tab %}}
+{{% tab name="Java" %}}
+
+```java
+@BelongsToContract(ExampleChainableContract.class)
+public final class ExampleChainableState extends ChainableState<ExampleChainableState> {
+
+  @Nullable
```

Comment:
you're not consistent with the same indentations ... please create 4 chars indentation like the prev file.

Reply:
You're still inconsistent in indentation - in 1 file it's 4 spaces and in other it's 2